tests/jobs/localisation/example_case/README

```diff
@@ -0,0 +1,44 @@
+## Directory for Test Case for Non-Adaptive Localisation (`LOCALISATION_JOB`)
+​
+### Main Components:
+​
+- **ERT config file:** `sim_field.ert`
+​
+  The ERT model depends on:
+  - `scripts/sim_fields.py`
+  - `scripts/common_functions.py`
+​
+​
+- **ERT Keyword Input:**
+  - **GRID:** Input grid is made by `scripts/init_test_case.py`
+  - **OBS_CONFIG:** Observations are extracted from an upscaled realization made by `scripts/init_test_case.py`
+  - **GEN_DATA:** Prediction of observables made by forward model `SIM_FIELD` which runs `scripts/sim_fields.py`
+  - **FIELD:** Prior realizations of field made by forward model `SIM_FIELD` which runs `scripts/sim_fields.py`
+​
+- **Other ERT Input Files:**
+  - `time_map.txt`
+  - `localisation.wf`
+​
+- **Other Files:**
+  - `randomseeds.txt`: Not used by ERT, but by `sim_fields.py`. This file can be generated by `scripts/init_test_case.py`
+  - `UpscaledGrid.EGRID`: Not used by ERT, but by `sim_fields.py`. This file can be generated by `scripts/init_test_case.py`
+​
+### Typical Workflow:
+​
+1. **Preparation:** Prepare ERT config input by running `scripts/init_test_case.py`.
+2. Directories for observations are created automatically according to the default settings.
+3. Make the directory `init_files` if not existing.
```

Comment:
Is step 3. needed given that we ran step 1.?

Also, we should give the `README` files the markdown extension `.md`.

Reply:
No, init_files directory is checked into the repo and has initially only the README file.
Fixed